kontrakter-felles/main/kotlin/no/nav/tilleggsstonader/kontrakter/felles/Tema.kt

```diff
@@ -23,4 +23,6 @@ fun Stønadstype.tilTema(): Tema =
         Stønadstype.BARNETILSYN -> Tema.TSO
         Stønadstype.LÆREMIDLER -> Tema.TSO
         Stønadstype.BOUTGIFTER -> Tema.TSO
+        Stønadstype.DAGLIG_REISE_TSO -> Tema.TSO
+        Stønadstype.DAGLIG_REISE_TSR -> Tema.TSR
```

Comment:
Må sjekke opp om vi faktisk skal ha to ulike temaer her, se kommentar øverst i fil